spec/requests/unpublishing_spec.rb

```diff
@@ -107,7 +107,6 @@
           schema_name: "redirect",
           base_path: base_path,
           publishing_app: edition.publishing_app,
-          public_updated_at: Time.zone.now.iso8601,
```

Comment:
I'm not sure this is related to the commit that changes it. I'm also guessing it has no effect since no tests needed changing - but could be wrong there?